products/microos/main.pm

```diff
@@ -37,7 +37,10 @@ sub load_boot_from_disk_tests {
     # Preparation for start testing
     loadtest 'microos/disk_boot';
     loadtest 'installation/system_workarounds' if is_aarch64;
-    loadtest 'transactional/enable_selinux'    if (get_var("ENABLE_SELINUX"));
+    replace_opensuse_repos_tests               if is_repo_replacement_required;
```

Comment:
is this applicable for SELinux case only? If so, should we wrap this in the same condition `if (get_var("ENABLE_SELINUX"))` ?

Reply:
For consistency I did the change unconditionally. IMO it's weird if the order is different just in the SELinux case